src/Sentry.NLog/SentryTarget.cs

```diff
@@ -0,0 +1,233 @@
+            {
+                _sdkDisposable = SentrySdk.Init(Options);
+            }
+        }
+
+        /// <inheritdoc />
+        protected override void Write(LogEventInfo logEvent)
+        {
+            if (logEvent?.Message == null)
+            {
+                return;
+            }
+
+            var hub = _hubAccessor();
+
+            if (hub?.IsEnabled != true)
+            {
+                return;
+            }
+
+            var exception = logEvent.Exception;
+            var formatted = Layout.Render(logEvent);
+            var template = logEvent.Message;
+
+            var contextProps = GetAllProperties(logEvent);
+
+            var shouldOnlyLogExceptions = exception == null && Options.IgnoreEventsWithNoException;
+
+            if (logEvent.Level >= Options.MinimumEventLevel && !shouldOnlyLogExceptions)
+            {
+                var evt = new SentryEvent(exception)
+                {
+                    Sdk =
+                    {
+                        Name = Constants.SdkName,
+                        Version = NameAndVersion.Version
+                    },
+                    Message = null,
+                    LogEntry = new LogEntry
+                    {
+                        Formatted = formatted,
+                        Message = template
+                    },
+                    Level = logEvent.Level.ToSentryLevel(),
+                    Release = Options.Release,
+                    Environment = Options.Environment,
+                };
+
+                evt.Sdk.AddPackage(ProtocolPackageName, NameAndVersion.Version);
+
+                // Always apply any manually configured tags
+                evt.SetTags(GetTags(logEvent));
+
+                if (IncludeEventProperties)
+                {
+                    evt.SetExtras(contextProps);
+                }
+
+                if (Options.SendEventPropertiesAsTags)
+                {
+                    evt.SetTags(GetLoggingEventProperties(logEvent).MapValues(a => a.ToString()));
+                }
+
+                hub.CaptureEvent(evt);
+            }
+
+            // Whether or not it was sent as event, add breadcrumb so the next event includes it
+            if (logEvent.Level >= Options.MinimumBreadcrumbLevel)
+            {
+                var message = string.IsNullOrWhiteSpace(formatted)
+                    ? exception?.Message ?? string.Empty
+                    : formatted;
+
+                IDictionary<string, string> data = null;
+
+                // If this is true, an exception is being logged with no custom message
+                if (exception != null && !message.StartsWith(exception.Message))
+                {
+                    // Exception.Message won't be used as Breadcrumb message Avoid losing it by adding as data:
+                    data = new Dictionary<string, string>
+                        {
+                            { "exception_data", exception.GetType().ToString() },
```

Comment:
Maybe exception_type is better? Since Exception.Data is actually a dictionary-property.

Reply:
Whoops, that's what I meant to do. Good catch. Made the changes then had some git issues and had to redo some things, guess I missed this issue.